src/libraries/System.DirectoryServices/src/System/DirectoryServices/DirectoryServicesPermission.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace System.DirectoryServices
 {
+#if CAS_OBSOLETIONS
```

Comment:
Why can't this be applied unconditionally? 

If this needs to be this way, maybe comment explaining why this is applied in src under ifdef, but missing in the reference assembly may be useful. It is not obvious to me.

Reply:
I saw that we had `_$(NetFrameworkCurrent)` as a target framework and I wasn't positive of the behavior of that. But talking with @safern last night, I understand it better. We are not actually compiling against .NET Framework there; this is for packaging purposes.  So you're right--I'll make it unconditional.  Thanks!

Reply:
@safern helped me understand the `ref` assembly aspect of `System.DirectoryServices` here too.  We need to start targeting `$(NetCoreAppCurrent)` in that ref assembly, which we didn't do before because the surface area had not yet diverged from `netstandard2.0`.  I'm going to address that as well.

Reply:
I can't get this working (today).  I'll need some infrastructure help getting `System.DirectoryServices` diverged so that its `ref` assembly can get built against `$(NetCoreAppCurrent)`.  Trying to do that led to transitive dependency challenges.

@GrabYourPitchforks and @eerhardt recommended we omit the DirectoryServices obsoletions from this PR and file an issue for RC1 to get the infrastructure updated to support adding them back.  I'm going to update this PR to reflect that and once that is looking solid, I'll submit an issue.

Reply:
I filed https://github.com/dotnet/runtime/issues/39401 for the infrastructure need. Assuming this PR gets merged before that is fixed, I'll file another tracking issue for marking the DirectoryServices types as obsolete.

Reply:
#39413 filed to mark those types as obsolete